packages/admin-app/src/index.tsx

```diff
@@ -19,10 +19,12 @@ ReactDOM.render(
     <React.StrictMode>
         <QueryClientProvider client={queryClient}>
             <IntlProvider locale="en" messages={en}>
```

Comment:
All of the language things will need to be put back at a later time.  Might need a new issue written for that.